docs/env_install.md

```diff
@@ -2,27 +2,25 @@
 ## Beacon setup
 
 1. [ Prerequisites ](#prerequisites)
-1. [ Installation ](#installation)
+1. [ Installation - on a virtual environment using pip](#installation-pip)
```

Comment:
I would suggest that you use a dependency manager like pipenv or poetry (I'm strongly for poetry after trying both for some time) to get deterministic builds. Right now there is no was to ensure that your tests and container will have exactly the same versions on all dependencies. But perhaps for another PR 😃

Reply:
I know I know :) In another PR!